<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>608</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Control Sales</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Sair">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Venda</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pesquise por:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela_Venda_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>671</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>107</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Código</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fabricante</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor unitário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor total</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>671</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Código</string>
         </property>
         <widget class="QLineEdit" name="lineCodVenda">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>19</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Nome</string>
         </property>
         <widget class="QLineEdit" name="lineNomeVenda">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Fabricante</string>
         </property>
         <widget class="QLineEdit" name="lineFabricanteVenda">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Pesquisar_venda">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total a pagar:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Total_a_pagar">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_Finalizar_Venda">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>400</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Finalizar venda</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>671</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Código</string>
         </property>
         <widget class="QLabel" name="label_Confirma_Cod">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>22</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Nome</string>
         </property>
         <widget class="QLabel" name="label_Confirma_None">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_13">
         <property name="title">
          <string>Fabricante</string>
         </property>
         <widget class="QLabel" name="label_Confirma_Fabricante">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Valor unitário</string>
         </property>
         <widget class="QLabel" name="label_Confirma_Valor_Und">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_RS">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>31</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Quantidade</string>
         </property>
         <widget class="QLineEdit" name="line__Confirma_Qtd">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Incluir_Venda">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Incluir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_Secreta_Cod">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela_Venda">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>671</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Código</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fabricante</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estoque</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor unitário</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Estoque</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tabela_Estoque">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Código</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fabricante</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantidade em estoque</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Ordenar por -&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordenar_Cod_Estoque">
           <property name="text">
            <string>Código</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordenar_Nome_Estoque">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordenar_Fabricante_Estoque">
           <property name="text">
            <string>Fabricante</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>661</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nome</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineNome"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Fabricante</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineFabricante"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Quantidade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineQtd"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Custo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineCusto"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Margem de venda</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineMargem"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Cadastrar">
         <property name="text">
          <string>Cadastrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tabela_Cadastro">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Código</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fabricante</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Custo</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Ordenar por -&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordena_Cod">
           <property name="text">
            <string>Código</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordena_Nome">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Ordena_Custo">
           <property name="text">
            <string>Custo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Relatório de vendas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>671</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Total de vendas</string>
         </property>
         <widget class="QLabel" name="label_Total_Vendas">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>181</width>
            <height>131</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Total de custos</string>
         </property>
         <widget class="QLabel" name="label_Total_custos">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>181</width>
            <height>131</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Total de lucros</string>
         </property>
         <widget class="QLabel" name="label_Total_Lucros">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>181</width>
            <height>131</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>671</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Média de venda por item</string>
         </property>
         <widget class="QLabel" name="label_Media_Vendas">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>301</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Média de lucro por item</string>
         </property>
         <widget class="QLabel" name="label_Media_Lucros">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>40</y>
            <width>301</width>
            <height>101</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionSalvar"/>
   </widget>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
